src/user-meta/framework/custom-meta/author-metadesc.php

```diff
@@ -36,4 +71,30 @@ public function get_field_id(): string {
 	public function is_empty_allowed(): bool {
 		return true;
 	}
+
+	/**
+	 * Renders the custom meta's field in the user form.
+	 *
+	 * @param int $user_id The user ID.
+	 *
+	 * @return void
+	 */
+	public function render_field( $user_id ): void {
+		echo '
+
+		<label for="' . \esc_attr( $this->get_field_id() ) . '">'
+			. \esc_html__( 'Meta description to use for Author page', 'wordpress-seo' )
+		. '</label>';
+
+		echo '
+
+		<textarea
+			rows="5"
+			cols="30"
+			id="' . \esc_attr( $this->get_field_id() ) . '"
+			class="yoast-settings__textarea yoast-settings__textarea--medium"
+			name="' . \esc_attr( $this->get_field_id() ) . '">'
+				. \esc_textarea( \get_the_author_meta( 'wpseo_metadesc', $user_id ) )
+		. '</textarea><br>';
+	}
```

Comment:
A couple things that I'm noticing here:
* Why did you choose for this to echo directly vs returning a string?
* Tying this rendering to the data feels wrong to me in this 🧅 land. Describing the data while also rendering the data seem to be separate goals. Might be overkill, but still I feel the need to mention this. How would this go if we switch the rendering to JS land for example?
* Using `.` in echo leads to unneeded concatenation, you can use `,` to continuing echo-ing
* Using hardcoded `wpseo_metadesc` should be replaced by the `get_key()`?
* Using method to retrieve the value in this output method (`get_the_author_meta` in this case), that sounds like an important method to have for the interface even?
* Why the extra whitespace and `<br>`?

Note I'm not repeating these thoughts for all the `render_field` methods 😉

Reply:
> Why the extra whitespace and `<br>`

My main goal here is to keep the rendered HTML exactly how it was before my PR, hence the extra whitespace and br tags.

Especially for whitespaces, I started my implementation with stripped whitespace and that had a direct corellation with a (tiny) visual regression in the UI, so this is the reason why

Reply:
I would think whitespace is squashed by the browser. Are you sure it is not just the `<br>`?

Reply:
nope, it was the whitespace 😁

Reply:
> Tying this rendering to the data feels wrong to me in this 🧅 land. Describing the data while also rendering the data seem to be separate goals. Might be overkill, but still I feel the need to mention this. How would this go if we switch the rendering to JS land for example?

I've laid my thoughts on this, on a prior comment, lets talk about it there :)

Reply:
> Why did you choose for this to echo directly vs returning a string?

It would make escaping and/or accounting for PHPCS issues a much harder task, if not outright impossible. 

We would have to basically do our escaping in the separate functions and then `phpcs:ignore` once when we echoed the final string, which would make this susceptible to security omissions.

Reply:
> Using . in echo leads to unneeded concatenation, you can use , to continuing echo-ing

The argument against unneeded concatenation is performance-related? Because I did what I did for readability purposes mostly, otherwise PHPCS complains.

I could do the following, to combine both methods, but I'm not sure if it's actually the best of two worlds:
```
		echo '

		<input
			class="yoast-settings__checkbox double"
			type="checkbox"
			id="', \esc_attr( $this->get_field_id() ), '"
			name="', \esc_attr( $this->get_field_id() ), '"
			aria-describedby="wpseo_content_analysis_disable_desc"
			value="on" '
			. \checked( \get_the_author_meta( 'wpseo_content_analysis_disable', $user_id ), 'on', false )
		. '/>';
```

So, if it's ok for you, I'd rather keep the current approach

Reply:
> * Using hardcoded `wpseo_metadesc` should be replaced by the `get_key()`?
> * Using method to retrieve the value in this output method (`get_the_author_meta` in this case), that sounds like an important method to have for the interface even?

Both good points, I'm implementing them together then.

Reply:
Got it, good context and it makes sense